var/spack/repos/builtin/packages/libidn2/package.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
```

Comment:
@Zehvogel the tests will fail until the boilerplate comment is stripped

Reply:
@scheibelp my bad, sorry, wonder how I missed that one. I hope everything looks ok now :)